pkg/config/config.go

```diff
@@ -1512,7 +1512,7 @@ func useHostEtc(config Config) {
 
 func checkConflictingOptions(config Config) error {
 	// Verify that either use_podman_logs OR docker_path_override are set since they conflict
-	if config.GetBool("logs_config.use_podman_logs") && config.IsSet("logs_config.docker_path_override") {
+	if config.GetBool("logs_config.use_podman_logs") && len(config.GetString("logs_config.docker_path_override")) > 0 {
```

Comment:
I assume this is why you were asking about how to "unset" something that has a default via `BindEnvAndSetDefault`?

Reply:
Yes. Was hoping to find a workaround for those impacted.